build.psm1

```diff
@@ -257,6 +257,32 @@ cmd.exe /C cd /d "$location" "&" "$($vcPath)\vcvarsall.bat" "$Arch" "&" cmake "$
             log "  Copying $srcPath to $dstPath"
             Copy-Item $srcPath $dstPath
         }
+
+        #
+        #   Build the ETW manifest resource-only binary
+        #
+        $location = "$PSScriptRoot\src\PowerShell.Core.Instrumentation"
+        Set-Location -Path $location
+
+        $command = @"
+cmd.exe /C cd /d "$location" "&" "$($vcPath)\vcvarsall.bat" "$Arch" "&" cmake "$overrideFlags" -DBUILD_ONECORE=ON -DBUILD_TARGET_ARCH=$Arch -G "$cmakeGenerator" . "&" msbuild ALL_BUILD.vcxproj "/p:Configuration=$Configuration"
+"@
+        log "  Executing Build Command for PowerShell.Core.Instrumentation: $command"
+        Start-NativeExecution { Invoke-Expression -Command:$command }
+
+        # Copy the binaries and manifest to the packaging directory
+        $dstPath = "$PSScriptRoot\src\powershell-win-core"
+        $FilesToCopy = @(
+            [IO.Path]::Combine($location, $Configuration, 'PowerShell.Core.Instrumentation.dll'),
+            [IO.Path]::Combine($location, 'PowerShell.Core.Instrumentation.man')
+            [IO.Path]::Combine($location, 'RegisterManifest.ps1')
```

Comment:
Do not bin-place the `.ps1` and `.man` file. Instead, we can copy them to `publish` folder directly by `powershell-win-core.csproj` file. See how we bin-place `install-powershellremoting` [here](https://github.com/PowerShell/PowerShell/blob/master/src/powershell-win-core/powershell-win-core.csproj#L22).

In this way, we don't need to add a `.gitignore` file in the `powershell-win-core` folder to ignore the `.man` file there.

Reply:
@daxian-dbw: Does this also mean they shouldn't be included in the nuget package?

Reply:
Fixed